### DISCUSSION TITLE
403 error from Netlify hosted feeds

### DISCUSSION THREAD

**Discussion by davidsneighbour** [gautamkrishnar/blog-post-workflow#239] [General]
This issue is the same as described in #113, this time at Netlify. The workaround is identical, using an RSS proxy service. The one mentioned in #113, rssmix.com, is not working anymore. I used https://rssproxy.migor.org/. I sent a note to the team at Netlify, but I don't expect them to be too accommodating, as it's on the free tier. 

[Issue report on the Netlify community forum](https://answers.netlify.com/t/loading-rss-feed-via-github-results-in-a-403-error/109625)

**Comment by gautamkrishnar** [comment]
You may need to whitelist the GItHub IP (https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/about-githubs-ip-addresses) addresses on Netlify. We can't do anything to fix this issue. Please contact netlfy support team to fix this issue.

**Comment by real-jiakai** [comment]
Last week, my gautamkrishnar/blog-post-workflow stopped working. The GitHub Action kept returning a 403 error, and my Gmail inbox was flooded with job failure notifications.

This evening, I looked into it. When I turned off Cloudflare's Bot Fight Mode in the security settings, then re-ran the GitHub Action job, it started working normally again.

![CleanShot 2024-07-20 at 20 12 52@2x](https://github.com/user-attachments/assets/4c84ce4a-0635-4362-88d3-95b939de01fd)

![CleanShot 2024-07-20 at 20 13 05@2x](https://github.com/user-attachments/assets/b80ee64b-48b1-403a-b47e-8f0d39e42230)

I hope my experience can be helpful to others who might encounter a similar issue.
